docs/source/roles/investigator/data_egress.md

```diff
@@ -1,7 +1,7 @@
 # Data egress process
 
-Data Safe Havens are used for doing secure data research on a single 'work package' of data.
-Once you have finished working with the data for your project, you'll have to egress any outputs of your work before the environment is shutdown.
+Data Safe Havens are used for doing secure data research.
+Once you have finished working with the data for your project, you'll have to conduct the egress any outputs of your work before the environment is shutdown.
```

Comment:
+1 for removal of ingress/egress as verbs, but it might take a while to do this universally!